tif  header reader changed?

Import test movies now fail with:
```
[   FAILED ] TestImportMovies.test_tif


Traceback (most recent call last):
  File "/home/gsharov/soft/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/home/gsharov/soft/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 676, in run
    self._callTestMethod(testMethod)
  File "/home/gsharov/soft/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 633, in _callTestMethod
    method()
  File "/home/gsharov/soft/scipion3/scipion-em/pwem/tests/protocols/test_protocols_import_movies.py", line 100, in test_tif
    self._checkOutput(protMovieImport, args, size=1, dim=(7676, 7420, 38))
  File "/home/gsharov/soft/scipion3/scipion-em/pwem/tests/protocols/test_protocols_import_movies.py", line 72, in _checkOutput
    self.assertEqual(dim, (x, y, n))
  File "/home/gsharov/soft/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 912, in assertEqual
    assertion_func(first, second, msg=msg)
  File "/home/gsharov/soft/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 1129, in assertTupleEqual
    self.assertSequenceEqual(tuple1, tuple2, msg, seq_type=tuple)
  File "/home/gsharov/soft/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 1100, in assertSequenceEqual
    self.fail(msg)
  File "/home/gsharov/soft/miniconda3/envs/scipion3/lib/python3.8/unittest/case.py", line 753, in fail
    raise self.failureException(msg)
AssertionError: Tuples differ: (7676, 7420, 38) != (7420, 7676, 38)
```

pwem.tests.data.test_data.TestImageHandler fails with the same error.

Both eman2 and imod report the header of data/tests/movies/c3-adp-se-xyz-0228_200.tif as 7676 x 7420

--- a/CHANGES.txt
+++ b/CHANGES.txt
@@ -1,3 +1,4 @@
+V3.5.1: hotfix tif reader dimensions
 V3.5.0
   Users:
      - add wait protocol. This protocol wait N seconds and then ends.

--- a/pwem/__init__.py
+++ b/pwem/__init__.py
@@ -44,7 +44,7 @@ from .objects import EMObject
 from .tests import defineDatasets
 from .utils import *
 
-__version__ = '3.5.0'
+__version__ = '3.5.1'
 NO_VERSION_FOUND_STR = "0.0"
 CUDA_LIB_VAR = 'CUDA_LIB'
 

--- a/pwem/emlib/image/image_readers.py
+++ b/pwem/emlib/image/image_readers.py
@@ -47,8 +47,9 @@ class TiffImageReader(ImageReader):
         tif = TiffFile(filePath)
         frames = len(tif.pages)  # number of pages in the file
         page = tif.pages[0]  # get shape and dtype of the image in the first page
-        x, y = page.shape
-        return x, y, frames, 1
+        x, y = page.imagewidth, page.imagelength  # IMPORTANT: to match xmipp convention
+
+        return x, y, 1, frames
 
 class EMANImageReader(ImageReader):
     """ Image reader for eman file formats"""

--- a/pwem/tests/data/test_data.py
+++ b/pwem/tests/data/test_data.py
@@ -241,9 +241,15 @@ class TestImageHandler(unittest.TestCase):
         tifffileLogger.disabled = True
 
         eerFile = self.dsFormat.getFile("eer")
-        logger.info("Reading dimension for %s" % eerFile )
+        logger.info("Reading dimension for %s" % eerFile)
         X, Y, Z, N = ih.getDimensions(eerFile)
         self.assertEqual([X, Y, Z, N], [4096, 4096, 1, 567])
+
+        # tif format
+        tifFile = self.dsFormat.getFile("c3-adp-se-xyz-0228_200.tif")
+        logger.info("Reading dimension for %s" % tifFile)
+        X, Y, Z, N = ih.getDimensions(tifFile)
+        self.assertEqual([X, Y, Z, N], [7676, 7420, 1, 38])
 
     def test_convertMicrographs(self):
         """ Convert micrographs to different formats.

